src/chain/electrum.rs

```diff
@@ -48,6 +51,48 @@ impl ElectrumRuntimeClient {
 		);
 		Ok(Self { electrum_client, bdk_electrum_client, tx_sync, runtime, logger })
 	}
+
+	pub(crate) async fn broadcast(&self, tx: Transaction) {
+		let electrum_client = Arc::clone(&self.electrum_client);
+
+		let txid = tx.compute_txid();
+		let tx_bytes = tx.encode();
+
+		let spawn_fut =
+			self.runtime.spawn_blocking(move || electrum_client.transaction_broadcast(&tx));
```

Comment:
General question - is the thread pool for spawn_blocking large enough for everything that ldk_node may require (concurrently)?

Reply:
Oh that's a lot. I had read somewhere it was 2*cpu and that seemed low.